### PR TITLE
Update about basic auth for Cloud version 10 projects

### PR DESCRIPTION
**Pull request by sajumb**

### CODE REVIEW

File: Umbraco-Cloud/Security/index.md [hunk 1]
```diff
@@ -76,7 +76,7 @@ By default, all ports are closed to secure them against external attacks. This i
 
 Umbraco Cloud offers a multitude of features allowing you to block access to different resources.
 
-- Basic Authentication allowing access to Backoffice & Frontend of Umbraco Cloud Websites only for authenticated users. **Note:** This feature is currently not available in Umbraco Cloud versions 9 and 10.
+- Basic Authentication allowing access to Backoffice & Frontend of Umbraco Cloud Websites only for authenticated users. **Note:** Basic authentication will not be available for Umbraco Cloud version 9 projects. It is available for Umbraco Cloud version 10 (and newer) versions, but the users are currently not able to exclude IP addresses for authentication using the allowlist feature.
```
Comment (jonat123) [inline Umbraco-Cloud/Security/index.md:79]:
```suggestion
- Basic Authentication allows access to the Backoffice & Frontend of Umbraco Cloud Websites for authenticated users only. 
:::note Basic authentication will not be available for projects running Umbraco 9. It is available for Umbraco Cloud version 10 (and newer) versions, however, the users are currently unable to exclude IP addresses for authentication using the allowlist feature.
:::
```

**Review by jonat123 — CHANGES_REQUESTED** [review]
@sajumb Looks good, I made some changes to the wording of the sentence as well as making the note in the format that we use on our so that it will look like a note :)

**Comment by sajumb** [comment]
Thanks. Suggestion approved! :)

File: Umbraco-Cloud/Security/index.md [hunk 2]
```diff
@@ -76,7 +76,9 @@ By default, all ports are closed to secure them against external attacks. This i
 
 Umbraco Cloud offers a multitude of features allowing you to block access to different resources.
 
-- Basic Authentication allowing access to Backoffice & Frontend of Umbraco Cloud Websites only for authenticated users. **Note:** This feature is currently not available in Umbraco Cloud versions 9 and 10.
+- Basic Authentication allows access to the Backoffice & Frontend of Umbraco Cloud Websites for authenticated users only. 
+:::note Basic authentication will not be available for projects running Umbraco 9. It is available for Umbraco Cloud version 10 (and newer) versions, however, the users are currently unable to exclude IP addresses for authentication using the allowlist feature.
```
Comment (jonat123) [inline Umbraco-Cloud/Security/index.md:80]:
```suggestion
:::note 
Basic authentication will not be available for projects running Umbraco 9. It is available for Umbraco Cloud version 10 (and newer) versions, however, the users are currently unable to exclude IP addresses for authentication using the allowlist feature.
```

**Comment by jonat123** [comment]
Merging it in

**Pull request merged**